tests/test_wiki_ES.py

```diff
@@ -9,27 +9,95 @@
 import responses
 
 @pytest.fixture(scope="session")
-def basket_ball_wiki_es(wiki_es_client, init_indices):
+def basket_ball_wiki_es(wiki_client, init_indices):
     """
     fill the elasticsearch WIKI_ES with a POI of basket ball
     """
     filepath = os.path.join(os.path.dirname(__file__), 'fixtures', 'basket_ball_wiki_es.json')
     with open(filepath, "r") as f:
         poi = json.load(f)
         poi_id = poi['id']
-        wiki_es_client.index(index='wikidata_fr',
+        wiki_client.index(index='wikidata_fr',
                         body=poi,
                         doc_type='wikipedia',
                         id=poi_id,
                         refresh=True)
         return poi_id
 
 @pytest.fixture(scope="session")
-def basket_ball(es_client):
+def basket_ball(mimir_client):
     """
     fill elasticsearch with a fake POI of basket ball
     """
-    return load_poi('basket_ball.json', es_client)
+    return load_poi('basket_ball.json', mimir_client)
+
+@freeze_time("2018-06-14 8:30:00", tz_offset=2)
+def test_undefined_WIKI_ES(basket_ball):
+    """
+    Check that when the WIKI_ES variable is not set
+    a Wikipedia call is observed
+    """
+    from idunn.blocks.wikipedia import WikidataConnector
+    client = TestClient(app)
+    WikidataConnector._wiki_es = None
+    wiki_es_ip = settings['WIKI_ES'] # temporary variable to store the ip of WIKI_ES to reset it after the test
+    settings._settings['WIKI_ES'] = None
+    with responses.RequestsMock(assert_all_requests_are_fired=False) as rsps:
+        rsps.add('GET',
+             re.compile('^https://.*\.wikipedia.org/'),
+             status=200,
+             json={"test": "test"})
+
+        for i in range(10):
+            response = client.get(
+                url=f'http://localhost/v1/pois/{basket_ball}?lang=fr',
+            )
+
+        assert len(rsps.calls) == 10
+    settings._settings['WIKI_ES'] = wiki_es_ip # put back the correct ip for next tests
```

Comment:
this won"t be done if the test fail, use a pytest fixture to do this

Reply:
indeed, done